src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/RazorProjectHost.cs

```diff
@@ -170,6 +187,25 @@ protected override Task InitializeCoreAsync(CancellationToken cancellationToken)
             });
         }
 
+        private async Task<string> DetectFallbackLanguageVersion()
```

Comment:
This is a much simplified version of what we had before... because we **only** care about the old supported scenarios of MVC1.X and MVC2.0.

Everything we care about in the future will use the MSBuild approach